src/main/java/org/ukma/spring/crooodle/dto/LoadRoomResponseDto.java

```diff
@@ -0,0 +1,4 @@
+package org.ukma.spring.crooodle.dto;
+
+public class LoadRoomResponseDto {
```

Comment:
Add all info relevant to the client here. Info relevant to hotel admins might be put in a diferrent dto that is going to be used in update and create